App/components/Image/image.styled.ts

```diff
@@ -1,17 +1,30 @@
 import styled from 'styled-components/native'
+import { Props } from './image'
+import { Border } from './enums/Border'
 
-export const ImageContainer = styled.TouchableOpacity`
-  background-color: ${({ theme }) => theme.colors.primary};
-  padding-vertical: 14;
-  border-radius: 6;
-  border-width: 1;
-  border-color: ${({ theme }) => theme.colors.primary};
-  margin-vertical: 7;
-`
+// export const ImageTry = styled.TouchableOpacity`
+//   background-color: ${({ theme }) => theme.colors.primary};
+//   padding-vertical: 14;
+//   border-radius: 6;
+//   border-width: 1;
+//   border-color: ${({ theme }) => theme.colors.primary};
+//   margin-vertical: 7;
+// `
 
-export const ImageText = styled.Text`
-  color: ${({ theme }) => theme.colors.white};
-  align-self: center;
-  font-size: 20px;
-  font-weight: 500;
+export const ImageContainer = styled.Image<Props>`
```

Comment:
Please consider adding ImageContainerProps since we don't use 'children' property here 😄